docs/options.md

```diff
@@ -193,3 +193,25 @@ By default, Prettier will wrap markdown text at the specified print width. In so
 | Default | CLI Override      | API Override        |
 | ------- | ----------------- | ------------------- |
 | `true`  | `--no-prose-wrap` | `proseWrap: <bool>` |
+
+## Indent Styles
+
+This indents adjacent CSS Rules based on their selectors naming conventions. This works well with popular CSS naming conventions such as [BEM](http://getbem.com/) or any naming convention that uses prefixes to indicate hierarchy.
+
+For example, a file with the following CSS will be indented as:
+
+```
```

Comment:
` ```css `?

Reply:
Unfortunately `css` was causing my Prettier plugin to flatten the example CSS (I have a feeling the CI rejected it too). So while it's not nicely syntax-highlighted CSS I'm guessing that I would need to change this later. Is that right?

Reply:
`<!-- prettier-ignore -->`